src/main/java/at/ac/tuwien/kr/alpha/grounder/parser/ParseTreeVisitor.java

```diff
@@ -543,14 +559,10 @@ public ExternalAtom visitExternal_atom(ASPCore2Parser.External_atomContext ctx)
 
 		List<Term> outputTerms = visitTerms(ctx.output);
 
-		return new ExternalAtom(
-			Predicate.getInstance(predicateName, outputTerms.size()),
-			interpretation,
-			visitTerms(ctx.input),
-			outputTerms
-		);
+		return new ExternalAtom(Predicate.getInstance(predicateName, outputTerms.size()), interpretation, visitTerms(ctx.input), outputTerms);
```

Comment:
I think the old formatting was done on purpose.

Reply:
fixed